backend/package.json

```diff
@@ -26,6 +27,7 @@
     "dotenv": "^16.0.3",
     "eslint-config-airbnb-base": "15.0.0",
     "express": "^4.18.2",
+    "fast-glob": "^3.2.12",
```

Comment:
This dependency was already being installed by other packages, so it won't affect our usage at all!